doc/source/formats/index.rst

```diff
@@ -2,8 +2,34 @@
 .. _formats-label:
 
 ====================
-Formats
+Format overview
 ====================
 
-So many Formats
+MDAnalysis can read topology or coordinate information from a wide variety of file formats. The emphasis is on formats used in popular computational chemistry software. By default, MDAnalysis figures out formats by looking at the extension, unless the format is :ref:`explicitly specified <universe-loading-label>` with the ``format`` or ``topology_format`` keywords.
 
+Below is :ref:`a table of formats in MDAnalysis <format-overview-label>`, and which information can be read from them. A topology file supplies the list of atoms in the system, their connectivity and possibly additional information such as B-factors, partial charges, etc. The details depend on the file format and not every topology file provides all (or even any) additional data.
+
+.. important ::
+
+    File formats do not always behave the way you might expect in MDAnalysis. It is *highly* recommended that you read the page for your data file format before using MDAnalysis. 
```

Comment:
This sounds as if MDAnalysis does things wrong. I think the truth is that file formats are ambiguous and we have to make choices that might not be the same that a user is used to. If we don't obey the published format then it's a bug.

I would rewrite as

File formats are complicated and not always well defined. MDAnalysis tries to follow published standards but this can sometimes surprise users. It is *highly* recommended that you read the page for your data file format instead of assuming certain behavior. If you encounter problems with a file format please [get in touch with us](https://lilyminium.github.io/UserGuide/contributing.html).

The last link should go to `Contributing`.